revealer_bot/launch_bot.py

```diff
@@ -1,24 +1,17 @@
 import os
 
 from bot_lair import the_actual_revealer_bot
-from message_parser import parse_message
+
 from revealer_bot.bob_and_other_networky_things import bob
 
 bot_token = os.environ["DISCORD_BOT_TOKEN"]
 
 
 @the_actual_revealer_bot.event
 async def on_ready():
-    print(f'Logged in as {the_actual_revealer_bot.user.name} ({the_actual_revealer_bot.user.id})')
-    print('------synced------')
-
+    print(f"Logged in as {the_actual_revealer_bot.user.name} ({the_actual_revealer_bot.user.id})")
+    print("------synced------")
 
-from revealer_commands import *
 
 the_actual_revealer_bot.run(bot_token)
 bob.start_learning_loop()
```

Comment:
Isn't the line 16 blocking?

Reply:
Hmm, Indeed it appears that we need to switch these two lines if we want to actually benefit from `bob.start_learning_loop()`